lyngk_board.js

```diff
@@ -0,0 +1,210 @@
+var lyngkModule = (function() {
+  my = {};
+  var verticalX = [1, 2, 3, 4, 5, 6, 7];
+  var verticalY1 = [3, 0, 1, 0, 1, 0, 3];
+  var verticalY2 = [9, 12, 11, 12, 11, 12, 9];
+  var diagonalSourceCoords = [[1, 3], [0, 6], [1, 7], [1, 9], [2, 10], [2, 12], [4, 12], [4, 0], [2, 0], [2, 2], [1, 3], [1, 5], [0, 6], [1, 9]];
+  var diagonalDestination = [[4, 0], [6, 0], [6, 2], [7, 3], [7, 5], [8, 6], [7, 9], [7, 3], [8, 6], [7, 7], [7, 9], [6, 10], [6, 12], [4, 12]];
+  var xRoot = Math.sqrt(3) * 25;
+  var yOffset = 25;
+  var padding = 25;
+  var boardPosition = {
+    A1: [0, 6],

```

Comment:
Can you put all the A's on a line, B's etc?  The longest lines may need breaking into two; hard to tell by inspection.